authui/src/authflowv2/icons/material-symbols.css

```diff
@@ -0,0 +1,37 @@
+@font-face {
+  font-family: "material-icons";
+  font-style: normal;
+  font-weight: 400;
+  src:
+    url("./icons/material-symbols-outlined.woff2") format("woff2"),
+    url("./icons/material-symbols-outlined.ttf") format("truetype");
+}
+
+.material-icons {
+  font-family: "material-icons";
+  font-weight: normal;
+  font-style: normal;
+  font-size: 24px; /* Preferred icon size */
+  display: inline-block;
+  line-height: 1;
+  text-transform: none;
+  letter-spacing: normal;
+  word-wrap: normal;
+  white-space: nowrap;
+  direction: ltr;
+
+  /* Support for all WebKit browsers. */
+  -webkit-font-smoothing: antialiased;
+  /* Support for Safari and Chrome. */
+  text-rendering: optimizeLegibility;
+
+  /* Support for Firefox. */
+  -moz-osx-font-smoothing: grayscale;
+
+  /* Support for IE. */
+  font-feature-settings: "liga";
+}
+
+.material-icons__passkey::before {
```

Comment:
Per offline discussion, let's use the ligatures feature documented in https://developers.google.com/fonts/docs/material_icons#using_the_icons_in_html then we can save our time declaring every icon we want to use here.